smarts/core/sumo_traffic_simulation.py

```diff
@@ -597,11 +597,6 @@ def _compute_traffic_vehicles(self) -> List[VehicleState]:
             speed = sumo_vehicle[tc.VAR_SPEED]
             vehicle_type = sumo_vehicle[tc.VAR_VEHICLECLASS]
             dimensions = VEHICLE_CONFIGS[vehicle_type].dimensions
-            # adjust sumo vehicle location if we shifted the map
-            map_offset = self._scenario.road_network.net_offset
-            assert len(map_offset) == 2
-            front_bumper_pos[0] += map_offset[0]
-            front_bumper_pos[1] += map_offset[1]
```

Comment:
To confirm, the reason why these are no longer needs is because when building the scenario, it is already using the shifted map thus traffic vehicle coordinates are shifted when built? Please correct me if otherwise. 

This part is also kinda needed when running third party simulators that add traffic directly to sumo right? if so, it is possible to have an configurable option in sumo_traffic_simulation to control shift coordinates or not.

Reply:
No, actually the reason is that, as of PR #727 , Sumo is started with the shifted map (thanks to the update to the `--net-file` parameter in `_base_sumo_load_params()` in `sumo_traffic_simulation.py`), so its coordinate system matches the coordinate system in use within Smarts. Vehicle locations coming from Sumo via Traci will be relative to the shifted/offset map already.

However, any third-party simulator that is adding vehicles to Sumo directly (outside of Smarts) may not know about the fact that the scenario map was shifted before Sumo was started. If they were to try to add vehicles to the map via Traci using the original coordinate system, things would seem off to them. So that's what necessitated adding the option to prevent map shifting entirely (i.e., not here, but where the scenario is built) in such situations.